modules/system/assets/ui/less/list.less

```diff
@@ -598,3 +598,11 @@ table.table.data {
         overflow: hidden;
     }
 }
+
+.list-scrollable-container.scroll-after {
+    th {
+        a {
```

Comment:
Please just turn this into one selector since there's only one rule: `.list-scrollable-container.scroll-after th a {`